[FP]: netty-tcnative-boringssl-static

### Package URl

pkg:maven/io.netty/netty-tcnative-boringssl-static@2.0.48.Final

### CPE

`cpe:2.3:a:chromium:chromium:2.0.48:*:*:*:*:*:*:*`
`cpe:2.3:a:chromium_project:chromium:2.0.48:*:*:*:*:*:*:*`

### CVE

- CVE-2011-1797
- CVE-2015-1346
- CVE-2015-1205
- CVE-2017-7000

### ODC Integration

{"label"=>"Maven Plugin"}

### ODC Version

7.0.0

### Description

As far as I understand from https://github.com/jeremylong/DependencyCheck/issues/4065, version `7.0.0` should contain already the fix for this case but it doesn't seem to be the case

```
[ERROR] Failed to execute goal org.owasp:dependency-check-maven:7.0.0:check (default) on project ***REDACTED***: 
[ERROR] 
[ERROR] One or more dependencies were identified with vulnerabilities that have a CVSS score greater than or equal to '7.0': 
[ERROR] 
[ERROR] netty-tcnative-boringssl-static-2.0.48.Final-osx-x86_64.jar: CVE-2015-1346(7.5), CVE-2011-1797(9.3), CVE-2017-7000(8.8), CVE-2015-1205(7.5)
[ERROR] 
[ERROR] See the dependency-check report for more details.
[ERROR] -> [Help 1]
```

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -4842,4 +4842,71 @@
         <cpe>cpe:/a:apache:tomcat</cpe>
         <cpe>cpe:/a:apache_tomcat:apache_tomcat</cpe>
     </suppress>
+    <suppress base="true">
+        <notes><![CDATA[
+       FP per issue #4129
+       ]]></notes>
+        <packageUrl regex="true">^pkg:maven/jakarta\.ws\.rs/jakarta\.ws\.rs-api@.*$</packageUrl>
+        <cpe>cpe:/a:eclipse:eclipse_ide</cpe>
+        <cpe>cpe:/a:oracle:java_se</cpe>
+        <cpe>cpe:/a:oracle:web_services</cpe>
+    </suppress>
+    <suppress base="true">
+        <notes><![CDATA[
+        FP per issue #4131
+        ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.graylog2/gelfclient@.*$</packageUrl>
+        <cpe>cpe:/a:graylog:graylog</cpe>
+    </suppress>
+    <suppress base="true">
+        <notes><![CDATA[
+       FP per issue #4133
+       ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.mariadb\.jdbc/mariadb-java-client@.*$</packageUrl>
+        <cpe>cpe:/a:mariadb:mariadb</cpe>
+    </suppress>
+    <suppress base="true">
+        <notes><![CDATA[
+        FP per issue #4135
+        ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.eclipse/yasson@.*$</packageUrl>
+        <cpe>cpe:/a:eclipse:eclipse_ide</cpe>
+        <cpe>cpe:/a:oracle:projects</cpe>
+    </suppress>
+    <suppress base="true">
+       <notes><![CDATA[
+       FP per issue #4140
+       ]]></notes>
+       <packageUrl regex="true">^pkg:maven/org\.aspectj/aspectjrt@.*$</packageUrl>
+       <cpe>cpe:/a:vmware:tools</cpe>
+    </suppress>
+    <suppress base="true">
+       <notes><![CDATA[
+       FP per issue #4149
+       ]]></notes>
+       <packageUrl regex="true">^pkg:maven/org\.apache\.kafka/kafka-log4j-appender@.*$</packageUrl>
+       <cpe>cpe:/a:apache:log4j</cpe>
+       <cpe>cpe:/a:apache:kafka</cpe>
+    </suppress>
+    <suppress base="true">
+       <notes><![CDATA[
+       FP per issue #4152
+       ]]></notes>
+       <packageUrl regex="true">^pkg:maven/org\.springframework\.cloud/spring-cloud-kubernetes-fabric8-config@.*$</packageUrl>
+       <cpe>cpe:/a:vmware:spring_cloud_config</cpe>
+    </suppress>
+    <suppress base="true">
+       <notes><![CDATA[
+       FP per issue #4156
+       ]]></notes>
+       <packageUrl regex="true">^pkg:maven/com\.lightbend\.akka\.management/akka-management-cluster-bootstrap_2\.13@.*$</packageUrl>
+       <cpe>cpe:/a:akka:akka</cpe>
+    </suppress>
+    <suppress base="true">
+       <notes><![CDATA[
+       FP per issue #4154
+       ]]></notes>
+       <packageUrl regex="true">^pkg:maven/io\.netty/netty-tcnative-boringssl-static@.*$</packageUrl>
+       <cpe>cpe:/a:chromium:chromium</cpe>
+    </suppress>
 </suppressions>

